src/main/java/fishstock/ui/DialogBox.java

```diff
@@ -0,0 +1,79 @@
+package fishstock.ui;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.HBox;
+import javafx.scene.paint.Color;
+
+/**
+ * This control represents a dialog box consisting of an ImageView to represent the speaker's face and a label
+ * containing text from the speaker.
+ */
+public class DialogBox extends HBox {
+    private static final CornerRadii CORNER_RADII = new CornerRadii(10);
+    private static final Background USER_DIALOG_BACKGROUND = new Background(
+            new BackgroundFill(Color.CHOCOLATE, CORNER_RADII, null));
+    private static final Background FISHSTOCK_BACKGROUND = new Background(
```

Comment:
Perhaps it would be better to use a common prefix instead (meaning `BACKGROUND_USER_DIALOG` and `BACKGROUND_FISHSTOCK` rather than the other way around)? This makes the two associated constants appear together when sorted alphabetically. I noticed the same issue in `Logic.java` too.